### PR TITLE
Laura/home view

### PR DESCRIPTION
**Pull request by lksevans12**
@smokecode @maggiedbaker @SarahCG Hey guys! I added the initial routes for '/' -- redirect to '/surveys' -- homepage and '/surveys/:name' for the single survey view. I also added the nav bar in layouts with and without session, and I added the index view page, however, since we don't have any survey's its showing up as an error because its trying to iterative over nil. 

I also added the helper methods in the auth.rb, and started a show page -- however, I'm not sure what we wanted in there, so I didnt know what to put in there besides the title. 

I also added the logged_in to the show route. Now, a user will need to log-in before viewing a single survey.

### CODE REVIEW

**Comment by maggiedbaker** [comment]
Thanks Laura! This is looking good. Just a couple things to change: 

- in the method logged_in?, please put a double bang (!!) in front of session[:user_id]. That changes the value into a boolean that will return true or false.
- Use the logged_in? and current_user helper methods in the rest of the code, rather than saying 'if session[:user_id]' or 'User.find(session[:user_id])', etc. That's what they're there for :)

**Comment by lksevans12** [comment]
Okay, made the changes. Does it look better?

**Comment by maggiedbaker** [comment]
Great, thanks! Just one tiny fix needed: current_user is a method, so remove the @ from it on line 25 of the layout. Then we should be all good! 

@lksevans12 @SarahCG @smokecode

**Comment by lksevans12** [comment]
ok!

**Comment by lksevans12** [comment]
okay!

**Comment by maggiedbaker** [comment]
Perfect!

File: app/controllers/surveys.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+get '/' do
+  redirect '/surveys'
+end
+
+get '/surveys' do 
+  @surverys = Survey.all
```
Comment (smokecode) [inline app/controllers/surveys.rb]: there's a typo on this line for surveys variable. @lksevans12

File: app/views/layout.erb [hunk 2]
```diff
@@ -18,6 +18,20 @@
   <title></title>
 </head>
 <body>
+
+<ul class = "navbar">
+<% if logged_in? %>
+  <li class = "bar"> <a href = "/surveys"> Surveys </a></li>
+  <li class="bar"><a href="/users/<%= session[:user_id] %>"><%= current_user.username %></a></li>
```
Comment (smokecode) [inline app/views/layout.erb]: can we comment out the link portion of this? We dont have that feature yet(or at all) and the username displaying is really cool. maybe "Yo," <username> ? or whatever is fun.

File: app/views/surveys/show.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<% @survey.title %>
```
Comment (smokecode) [inline app/views/surveys/show.erb]: did you want to display this? missing an equal sign if so. @lksevans12

**Pull request merged**